doc/guide/packages.xml

```diff
@@ -128,19 +114,45 @@ $ cockpit-bridge --packages
           string version number. The package will only be usable if the Cockpit bridge
           and javascript base are equal or newer than the given version number.</para></listitem>
       </varlistentry>
-      <varlistentry>
-        <term>tools</term>
-        <listitem><para>An optional JSON object containing all the tools that this package
-          provides. These will be added into the Cockpit user interface under the 'Tools' menu.
-          Each property on this object is named for the identifier of such a tool, and the
-          property value is a JSON object described below.</para></listitem>
-      </varlistentry>
       <varlistentry>
         <term>version</term>
         <listitem><para>An informational version number for the package.</para></listitem>
       </varlistentry>
     </variablelist>
 
+    <para>In addition, the following keys contain information about where components of the package
+      should appear in Cockpit's user interface. Each of these keys is optional and contains an
+      object mapping unique identifiers to menu items, which are described below. (The naming of
+      these fields doesn't perfectly match the current user interface for historical reasons.)
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>dashboard</term>
+        <listitem><para>Dashboard items appear in the very left navigation bar, next to Cockpit's
+        <emphasis>Dashboard</emphasis> and the machines themselves.</para></listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>menu</term>
+        <listitem><para>These items appear in the sidebar for an individual machine, next to
+          <emphasis>System</emphasis>, <emphasis>Logs</emphasis>, etc.  This section is roughly
+          ordered into these categories:
+          <itemizedlist>
+            <listitem><para>System Information (10)</para></listitem>
+            <listitem><para>Logs (20)</para></listitem>
+            <listitem><para>Configuring major subsystems (30-40)</para></listitem>
+            <listitem><para>Things running on the machine (VMs, Containers - 50-60)</para></listitem>
+            <listitem><para>Implementation Details (Accounts, Services - 70-100)</para></listitem>
```

Comment:
Is it clear that these numbers are priority? Should we state that above the list?

Perhaps something like this: "These items appear, along with their priority number, in the sidebar…"

Reply:
Right. I went with `order`, to be consistent with the field in the menu item object that's documented below.